### ISSUE TITLE
Website down?

### ISSUE THREAD

**Issue by cpp77** [drewnoakes/fix-decoder#44]
Hi, used your website years ago. However, it doesn't seem to be up anymore?

**Comment by drewnoakes** [comment]
Yep the site's down at the moment. I hope to get it back up and running before too long, but life's getting in the way.

**Comment by cpp77** [comment]
Thanks for replying. Ah, awesome. I've just had a look for substitutes and I can comfortably say your's is the best.

**Comment by chrismasters** [comment]
YMMV but if you use Docker, here's a Dockerfile so you can just run it yourself without worrying about building. It pulls the repository, builds it and exposes the app on localhost:8080

I would offer this up as a pull request but not sure it really belongs in the repo. Maybe in the docs?

- Save somewhere as Dockerfile
- Run `docker build -t fix-decoder .`
- Run the container `docker run -p 8080:8080 fix-decoder`
- Browse to http://locahost:8080

```dockerfile
# Use an official Node.js image
FROM node:16-alpine

# Set the working directory inside the container
WORKDIR /app

# Install git and other necessary packages
RUN apk add --no-cache git

# Clone the fix-decoder repository from GitHub
RUN git clone https://github.com/drewnoakes/fix-decoder.git .

# Install Gulp globally
RUN npm install -g gulp

# Install project dependencies using Yarn
RUN yarn install

# Build the project using Gulp
RUN gulp

# Install a simple HTTP server to serve the static files from the 'dist' directory
RUN npm install -g http-server

# Expose port 8080 for the HTTP server
EXPOSE 8080

# Command to start the HTTP server, serving from the 'dist' folder
CMD ["http-server", "dist", "-p", "8080"]
```

**Comment by drewnoakes** [comment]
Thanks, that's awesome. I expect I'd use Azure Static Apps to host it, but because I don't have a dedicated domain name for it, and everything under drewnoakes.com is offline right now, I expect I'll bring the whole domain back online in one hit. Maybe I could do this with a subdomain though, like `fixdecoder.drewnoakes.com`.

**Comment by AndrewJD79** [comment]
Yeah, it was the best decoder I've seen so far.
